src/api/WebLink.js

```diff
@@ -28,6 +28,44 @@ class WebLink extends Item {
         const suffix: string = id ? `/${id}` : '';
         return `${this.getBaseApiUrl()}/web_links${suffix}`;
     }
+
+    /**
+     * Gets a box weblink.
+     *
+     * @param {string} id - Weblink id
+     * @param {Function} successCallback - Function to call with results
+     * @param {Function} errorCallback - Function to call with errors
+     * @param {Array<String>} fields - Array of field strings
+     * @returns {Promise}
+     */
+    async getWeblink(
+        id: string,
+        successCallback: Function,
```

Comment:
Can we use stronger types for the success and error callbacks?

Reply:
sure